packages/tasit-contracts/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "tasit-contracts",
+  "version": "0.0.1",
+  "description": "",
+  "keywords": [
+    "ethereum",
+    "dapp",
+    "truffle",
+    "ganache",
+    "solidity"
+  ],
+  "author": "Tasit Labs",
+  "homepage": "https://github.com/tasitlabs/TasitSDK#readme",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/tasitlabs/tasitsdk.git"
+  },
+  "scripts": {
+    "ganache:start": "npx ganache-cli -m 'beach swap combine paper music cook electric bullet trust actress liquid asthma' > /dev/null 2>&1 &",
```

Comment:
What degree of customization is here in the truffle and ganache scripts? Is it straight from the truffle docs? Or was this pulled from a tutorial somewhere? Just curious. Thanks!

Reply:
It was the way that I've founded to run ganache in the background and quiet its screen output easily. 
Reading again truffle scripts seems to me that run it in the background is unnecessary. I'll review these scripts.

Reply:
I'm not sure about run ganache-cli via truffle. I've never seen this kind of setup. As ganache is used to replace blockchain, I don't know if is the case to start a blockchain from a dev/deploy tool. I've done a quick research about how others projects are automatizing truffle/ganache, and I've seen many of them that are using Docker to start/stop ganache (with very similar logic that we are doing). What do you think?

Reply:
I think I may have been thinking about TestRPC vs ganache-cli as opposed to using truffle vs. ganache. https://github.com/trufflesuite/ganache-cli/blob/master/README.md

I'm going to close this thread out. Thanks!